### PR TITLE
Re-structuring of chapters, changes in headings

### PR DESCRIPTION
**Pull request by clara-rthrdt**
Introducing a new chapter structure based on the UI/UX feedback we have received.

### CODE REVIEW

File: docs/docs/01_get-started/_category_.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+da{
```
Comment (cathleenberger) [inline docs/docs/01_get-started/_category_.json:1]:
```suggestion
{
```

**Comment by chartgerink** [comment]
Hi 👋 I was asked by @cathleenberger to see whether I could resolve the conflicts. It seems there are (at least) 17 conflicts when I tried this. My recommendation would be to create a new branch from `main` and open a new pull request. 😊

**Comment by cathleenberger** [comment]
Thank you very much for your support. Following Chris' advice, I'm closing this PR to resolve it. Would you be able to reintroduce this in a new branch?

**Comment by clara-rthrdt** [comment]
Thanks for your support @chartgerink, I will make my suggestions in a new branch and open another PR!

**Pull request closed without merging**